### ISSUE TITLE
[Popover] Popover 在 trigger hover 下 如果预先点击 trigger 而不是 hover，会导致浮层内元素无法触发 click 事件

### ISSUE THREAD

**Issue by DaiQiangReal** [DouyinFE/semi-design#875]
### Which Component 出现bug的组件
  - Popover

### semi-ui version
  - latest
    
### Expected result 期望的结果是什么
  - 内部元素正常接收 click 事件

### Actual result 实际的结果是什么
  - 浮层消失

### Steps to reproduce 复现步骤

If your reproduction scene is complicated, you can list the reproduction steps here

如果你的复现场景比较复杂，可以在这里列出复现的步骤
  
 - 先点击 Button，然后会发现无法点击 popover 内的 select

### Reproducible code 复现代码

- You can paste a minimal reproducible code, preferably runnable in the code editing area of [Semi official website](https://semi.design/en-US/input/button)

```
import React from 'react';
import { Popover, Tag } from '@douyinfe/semi-ui';

function Demo() {
    return (
        <Popover
            content={
                <Select defaultValue='abc' style={{ width: 120 }}>
            <Select.Option value='abc'>抖音</Select.Option>
            <Select.Option value='hotsoon'>火山</Select.Option>
            <Select.Option value='jianying' disabled>剪映</Select.Option>
            <Select.Option value='xigua'>西瓜视频</Select.Option>
        </Select>
            }
        >
            <Button>点击此处</Button>
        </Popover>
    );
}

```

- You can also use Codesandbox to create a minimal reproduction scene use Semi [template](https://codesandbox.io/s/semi-design-simple-story-oubej?file=/src/App.jsx)

Reproducible Codesandbox link: https://codesandbox.io

- Or, if your scene is more complex, you can use other templates to create a reproducible scene

Reproducible Codesandbox link: https://codesandbox.io

### Additional information 补充说明
- 遇到这个bug的业务场景、上下文、或者你的需求场景

**Comment by shijiatongxue** [comment]
it's a bug related to a11y. Trigger now listen blur event when `trigger` is hover(default), maybe we should judge whether `e.target` in content or trigger firstly.

**Comment by YannLynn** [comment]
fixed in 2.17.0-beta.0
